DistributedResponseCachingMiddleware/Utility/CustomResponseCachingPolicyProviderExtensions.cs

```diff
@@ -0,0 +1,28 @@
+﻿using Microsoft.AspNetCore.ResponseCaching;
+using Microsoft.Extensions.DependencyInjection;
+using System;
+
+namespace ExternalNetcoreExtensions.Utility
+{
+    /// <summary>
+    /// Extension methods for the ResponseCaching Policies.
+    /// </summary>
+    public static class CustomResponseCachingPolicyProviderExtensions
+    {
+        /// <summary>
+        /// Add policy that allows caching even if it exists an Authorization header.
```

Comment:
Grammar: even if an Authorization header exists
(This is more a note - I'll fix grammer things before I merge this in if you haven't already)

Reply:
I've made a commit  to solve the above issues, please check it.